File: android/src/main/java/com/skjline/rideraid/ui/preferences/MainPreferenceActivity.java
package com.skjline.rideraid.ui.preferences;

import android.os.Bundle;
import android.preference.PreferenceActivity;

import com.skjline.rideraid.R;

public class MainPreferenceActivity extends PreferenceActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        getFragmentManager().beginTransaction().replace(android.R.id.content, new PreferenceFragment()).commit();
    }

    public static class PreferenceFragment extends android.preference.PreferenceFragment {
        @Override
        public void onCreate(final Bundle savedInstanceState) {
            super.onCreate(savedInstanceState);

            addPreferencesFromResource(R.xml.preferences);
        }
    }
}

File: rider/src/main/java/com/pss9/rideraid/ui/preferences/MainPreferenceActivity.kt
package com.pss9.rideraid.ui.preferences

import android.os.Bundle
import android.preference.PreferenceActivity
import com.pss9.rideraid.R

class MainPreferenceActivity : PreferenceActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        fragmentManager.beginTransaction().replace(android.R.id.content, PreferenceFragment()).commit()
    }

    class PreferenceFragment : android.preference.PreferenceFragment() {
        override fun onCreate(savedInstanceState: Bundle?) {
            super.onCreate(savedInstanceState)

            addPreferencesFromResource(R.xml.preferences)
        }
    }
}
